apps/functions-subscription/local.settings.json.example

```diff
@@ -5,10 +5,18 @@
     "NODE_ENV": "development",
     "COSMOSDB_ENDPOINT": "<put-here-the-the-cosmos-database-endpoint>",
     "COSMOSDB_DATABASE_NAME": "<put-here-the-name-of-cosmos-database>",
+    "COSMOSDB_CONTAINER_NAME_LEASES": "leases",
```

Comment:
**note**: Be consistent with https://github.com/pagopa/trial-system/pull/52/files#diff-5a819a9ec7cfcd059b2355234847b8a89ca0e7b77ff750565537eccb077f90b2R15